integration/Dunet.Integration.csproj

```diff
@@ -9,7 +9,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Dunet" Version="1.10.0" />
+        <PackageReference Include="Dunet" Version="1.11.0" />
```

Comment:
```suggestion
        <PackageReference Include="Dunet" Version="1.10.0" />
```
Let's not bump this right now or it won't build in GH actions.

Reply:
I know you probably bumped this for testing purposes, so consider this a reminder to revert before we can merge :)

Reply:
Thanks, I've reverted that :D